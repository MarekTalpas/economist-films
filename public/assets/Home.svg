<?xml version="1.0" encoding="UTF-8"?>
<svg width="60px" height="60px" viewBox="0 0 60 60" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
    <!-- Generator: Sketch 42 (36781) - http://www.bohemiancoding.com/sketch -->
    <title>Home</title>
    <desc>Created with Sketch.</desc>
    <defs></defs>
    <g id="Symbols" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
        <g id="Side-Panel" transform="translate(-20.000000, -370.000000)" fill="#FFFFFF">
            <g id="Home">
                <path d="M70,399.220779 L70,430 L30,430 L30,399.220779 L20,399.220779 L50,370 L80,399.220779 L70,399.220779 Z M44,414.415584 L44,430 L56,430 L56,414.415584 L44,414.415584 Z"></path>
            </g>
        </g>
    </g>
</svg>