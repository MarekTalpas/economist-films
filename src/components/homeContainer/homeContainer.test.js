// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import Home from './homeContainer';

test('homeContainer renders correctly', () => {
  const tree : string = renderer.create(<homeContainer />).toJSON();
  expect(tree).toMatchSnapshot();
});
