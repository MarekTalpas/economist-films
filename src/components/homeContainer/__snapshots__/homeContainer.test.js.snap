exports[`test HomeContainer renders correctly 1`] = `<div />`;
